src/dftbp/timedep/timeprop.F90

```diff
@@ -2459,6 +2487,14 @@ subroutine writeTDOutputs(this, dipoleDat, qDat, energyDat, forceDat, coorDat, f
       end if
     end if
 
+    if (this%tWriteAtomEnergies) then
+        write(atomEnergyDat, "(2X,2F25.15)", advance="no") time * au__fs, energy%Etotal
+        do iAtom = 1, this%nAtom
+          write(atomEnergyDat, "(F25.15)", advance="no")this%energy%atomTotal(iAtom)
+        end do
+      write(atomEnergyDat, *)
```

Comment:
Wouldn't it be preferable to adopt tagged output here?

Reply:
Why tagged output?

Reply:
So it's a) easily machine readable, b) can be checked for changes/bugs by the regression framework and c) if the tagged process ever gets co-opted to redirect through the API this automatically returns your data that way as well.

Reply:
I see, but I thought Sebastian meant to replace the current output file by a tagged one, is that what he meant? Because for regression test purposes I can just add an entry to autotest.tag with this info

Reply:
`csv` is an easy machine readable format (which is also compatible with list directed reads).

Reply:
I guess the question is how large this data is going to get, presumably its written at most iterations? For regression purposes, a snapshot of the final set would be OK.
But for machine readable data in real calculations, I think we need to set up a proper framework (including generating CSV or similar as an option or binary once things get big). That would be outside the scope of this PR.

Reply:
For this particular case I don't expect the data to be huge, it's only Natoms x Noutputsteps, and by default the output frequency is 50 steps, Noutputseps = Nsteps / 50. We have massive outputs like the bond-wise populations or energy that are stored in binary files, but for other outputs like orbital populations (which is Norbs x Noutputsteps) I guess it would be good to set such a framework.